apps/core-app/src/pages/NewProposal.tsx

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+import { FormBuilder } from '@daohaus/haus-form-builder';
+
+import { getFormLegoById } from '../legos/form';
+import { CustomFields } from '../legos/config';
+
+export function NewProposal() {
+  const location = useLocation();
+
+  const formLego = useMemo(() => {
+    const params = new URLSearchParams(location.search);
+    const legoId = params.get('formLego');
+    if (!legoId) return null;
+    return getFormLegoById(legoId);
+
+    // const legoBase64 = params.get('formLego');
```

Comment:
clean

Reply:
Or is this for use later?

Reply:
yeah - didn't want to lose it - but i can make a ticket and clean this out.